donate/templates/partials/donate-form.html

```diff
@@ -0,0 +1,145 @@
+<div class="donate-form">
+    <header class="donate-form__header">
+        {% if title %}
+            <h1 class="heading heading--primary heading--bottom-margin">{{ title }}</h1>
+        {% endif %}
+        <div class="donate-form__currency">
+            <label for="currency-select" class="donate-form__currency-label">Currency</label>
+            <select autocomplete="off" class="donate-form__currency-dropdown" id="currency-select">
+                <option selected="" value="usd">USD &nbsp; $</option>
```

Comment:
We'll generate this dropdown from the back-end eventually, using data from here: https://github.com/mozilla/donate-wagtail/pull/37/files#diff-8754effadd5f2a07a8fcb931223f5c8e